### DISCUSSION TITLE
Toast - Weird display issue, the toast message display at both the top & bottom right for the first time the form is displayed.

### DISCUSSION THREAD

**Discussion by lcallif** [livewire/flux#864] [Feature Requests]
The first time a toast action is performed the toast displays at the top and bottom right.  The 2nd and subsequent times it displays at both the top & bottom right.  I thought that I may have added an extra toast message display, but then I found that as long as I don't redisplay the form I only see the bottom right toast message.  I would put this in as an Issue, but I am not 100% sure that I didn't miss something.

I am not worried about this, just curious as to why.  The following is the code I am using.

Livewire Component
```
      use Flux\Flux;
      .....
      .....
      .....
      .....
      Flux::toast('Congratulations, You Updated Lead: ' . $this->nickname);
```

Livewire Blade
```
    @persist('bct-toast')
          <flux:custom.bct-toast />
    @endpersist
```

**Comment by jeffchown** [comment]
@lcallif When calling `Flux::toast` from within your Livewire component, you need to make sure you have the Flux Toast code loaded on the page - if you will be using `Flux::toast` in many components/places, most times it makes sense to put it in your layout file just before the `<body>` tag:

```
    @persist('toast')
        <flux:toast />
    @endpersist
</body>
```

See docs: https://fluxui.dev/components/toast

So when you then use/render your Livewire components in a view, any calls to `Flux::toast(...)` in those components' code will be able to display the toaster.

**Reply by lcallif** [reply]
I did that, but I must have missed one errant toast display.

I will check again and report back on my findings.  

Ok, I will close this and reopen it if I don't find the duplicate...  Dang.
